cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -81,6 +81,24 @@ def ensure_vm_running(small_vm):
         raise Exception("Unknown power state - unable to continue!")
 
 
+@pytest.fixture(params=["stopped", "running"])
```

Comment:
I would stay on the same terms as other tests are using: cold and hot, what do you think?